### DISCUSSION TITLE
Is it possible to set your working directory independent of your output directory?

### DISCUSSION THREAD

**Discussion by CrumpThePenguin** [mifi/lossless-cut#2195] [Q&A]
If I go into the settings and set "Output Directory" it works fine, but as soon as I go to export a clip it seems to change that directory. Is it not possible to have all the proj.llc files in their own folder, and my output files also in their own folder? It just get cluttery when I have 20 different clips and each has its own .llc next to it, especially as once I'm done exporting I almost never go back into the source file for more editing.

I'm probably missing some interaction with settings so if someone can help me figure this out that'd be swell.

**Comment by mifi** [comment]
by "output directory" i guess you mean "Working directory". There's a setting called "Store project file (.llc) in the working directory or next to loaded media file". you can set it to "store next to media file". if you don't want .llc files at all you can turn off the setting "Auto save project file"
